storage-bigtable/src/proto/confirmed_block.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
```

Comment:
I think I'd prefer this layout:
```
storage-bigtable/src/confirmed_block.proto
storage-bigtable/proto/confirmed_block.rs
```

1. keeping generated files out of `src/` in general feels best, as we unleash linters and formatters on `src/`.  
2. No need for `src/proto/blah.proto` in this case, `src/blah.proto` is just as good since the number of files in this crate is small
3. I'm not a superfan of `storage-bigtable/proto/...` as the location for generated protobuf source files, but I guess it works and it's not `src/`